test/runtime-escaping-test.js

```diff
@@ -39,12 +39,18 @@ describe('Content escaping', function() {
     '<unescaped>');
   });
 
-  it('should ignore other props if html exists', function() {
+  it('should ignore html field if block/elem/cls/attrs/tag exists', function() {
     test(function() {
     }, { block: 'b', html: '<unescaped>', content: 'safe text' },
     '<div class="b">safe text</div>');
   });
 
+  it('should render html if only content and html fields exists', function() {
```

Comment:
I'd say: `it should determine unescaped html field if there are no bem-entity properties`.